src/Wallabag/CoreBundle/Resources/views/themes/material/Config/index.html.twig

```diff
@@ -183,6 +184,153 @@
                         </form>
                     </div>
 
+                    <div id="set5" class="col s12">
+                        <div class="row">
+                            <div class="input-field col s12">
+                                <ul>
+                                    {% for tagging_rule in app.user.config.taggingRules %}
+                                    <li>
+                                        if « {{ tagging_rule.rule }} » then tag as « {{ tagging_rule.tags|join(', ') }} »
+                                        <a href="{{ path('delete_tagging_rule', {id: tagging_rule.id}) }}" title="{% trans %}Delete{% endtrans %}">
+                                            <i class="tool grey-text delete mdi-action-delete"></i>
+                                        </a>
+                                    </li>
+                                    {% endfor %}
+                                </ul>
+                            </div>
+                        </div>
+
+                        <form action="{{ path('config') }}#set5" method="post" {{ form_enctype(form.new_tagging_rule) }}>
```

Comment:
Could you use `form_start(form.new_tagging_rule)` instead of this line? Mostly because `form_enctype` will be removed in Symfony3.0